### ISSUE TITLE
Posenet Model Does Not Load Suddenly

### ISSUE THREAD

**Issue by pinglynn** [ml5js/ml5-next-gen#161]
Hello, my team and I, who are new to coding, have been using visual studio code to run our Posenet model. The model has been working well and our code runs smoothly. 

However, just recently, after running our code, it continues showing "Loading Model..." and the model does not show up. This happened all of a sudden and we are unable to find the cause of this. This issue has not occurred before despite using the exact same codes. There are no errors in the code and the only error that we can find anywhere at all is on the console of the loading model which states that the source "ml5.posenet" in our code is not a function. 

We have tried doing research but due to our lack of knowledge, we find it difficult to find solutions. Please advise us on how we can move forward. If there are any questions that we can help clarify, please let us know. Thank you.

**Comment by shiffman** [comment]
Hi! Did you figure it out? You probably need to make sure you are referencing the older version of the library!

We released a major 1.0 update and all the models are new! Your code should still work by referencing ml5@0.12.2 in index.html rather than ml5@latest! 

```html
    <script src="https://unpkg.com/ml5@0.12.2/dist/ml5.min.js"></script>
```

When you want to try the new release:

```html
    <script src="https://unpkg.com/ml5@1/dist/ml5.min.js"></script>
```

**Comment by pinglynn** [comment]
I see, thank you so much!
